voice/bxml/callbacks/answer.md

```diff
@@ -39,6 +39,7 @@ POST http://[External server URL]
 ```json
 {
 	"eventType"     : "answer",
+    "eventTime"     : "2019-06-20T15:54:25.435Z",
 	"accountId"     : "55555555",
```

Comment:
Misalignment (happens other places too)

Reply:
I don't understand, these are aligned in my IDE. I will ping you about it tomorrow.